NitroxModel/DataStructures/GameLogic/VehiclePlayer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ProtoBuf;
+using System;
+using UnityEngine;
+
+namespace NitroxModel.DataStructures.GameLogic
+{
+    [Serializable]
+    [ProtoContract]
+    public class VehiclePlayer
```

Comment:
This looks to be unused.  If it is not, why do we need it when it only has one string member?

Reply:
hahaha i forget create that xD